src/app/core/services/github.service.ts

```diff
@@ -483,8 +483,8 @@ export class GithubService {
     pluckEdges: (results: ApolloQueryResult<T>) => Array<any>,
     Model: new (data) => M
   ): Observable<Array<M>> {
-    return from(this.withPagination<T>(pluckEdges)(query, variables)).pipe(
-      map((results: Array<ApolloQueryResult<T>>) => {
+    return this.withPagination<T>(pluckEdges, query, variables).pipe(
```

Comment:
```suggestion
    return this.withPagination<T>(pluckEdges, query, variables, false).pipe(
```

Reply:
I've added this, I initially misunderstood your previous comment, sorry!